tests/backup/backup_basic_test.go

```diff
@@ -254,9 +254,23 @@ var _ = AfterSuite(func() {
 				}
 				err = DeleteCluster(clusterName, orgID, ctx, true)
 				Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cluster %s", clusterName))
-				if clusterCredName != "" {
-					err = DeleteCloudCredential(clusterCredName, orgID, clusterCredUID)
-					Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Verifying deletion of cluster cloud cred [%s]", clusterCredName))
+				if err == nil {
```

Comment:
Why do you have to check for err?
We can do the same thing which you are doing in the other cleanup right?

```
				clusterDeleteStatus := func() (interface{}, bool, error) {
					status, err := IsClusterPresent(clusterName, ctx, orgID)
					if err != nil {
						return "", true, fmt.Errorf("cluster %s still present with error %v", clusterName, err)
					}
					if status {
						return "", true, fmt.Errorf("cluster %s is not deleted yet", clusterName)
					}
					return "", false, nil
				}
				_, err = task.DoRetryWithTimeout(clusterDeleteStatus, clusterDeleteTimeout, clusterDeleteRetryTime)
				Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Deleting cluster %s", clusterName))
```

Reply:
Yes correct. Don't know how I missed it.
Corrected now